validation/score.py

```diff
@@ -26,10 +31,12 @@ def parse_args():
         help="Path to the validation CSV file",
     )
     parser.add_argument(
-        "--latitude-resolution", default=0.005, type=float, help="Latitude resolution"
```

Comment:
question: why are we getting rid of this arg?

Reply:
I'm abandoning this sampling method because it was convoluted and doesn't play well with the segments